### DISCUSSION TITLE
How to hide left panel on startup in Zed Preview 0.128.1?

### DISCUSSION THREAD

**Discussion by failable** [zed-industries/zed#9630] [Feature Requests]
After upgrading to 0.128.x, quitting Zed then restart or opening new projects / switching projects the left panel is always shown. How can I stop that? Thanks.

**Comment by Moshyfawn** [comment]
I believe it remembers your last state when you last closed the project. I don't think there is a default state setting for the Project panel at the moment.

**Reply by failable** [reply]
It doesn't. I hide the pane, then close the project / quit Zed, then open the project / start Zed, the pane is there.

**Reply by failable** [reply]
It does in the previous versions (<= 0.127.x).

**Reply by Moshyfawn** [reply]
Interesting. I can't seem to reproduce it. How do you toggle the pane: via the keyboard shortcut or via the UI button? Do you have any custom settings specific to the project panel?

**Reply by failable** [reply]
I use `"ctrl-cmd-l": "workspace::ToggleLeftDock"` and press `ctrl-cmd-l` to close it. 

----

Updated: I tried the menu bar (`View - Toggle Left Dock`), the result is same.

I don't have any project specific settings. And it happens on all projects.

**Reply by Moshyfawn** [reply]
Upgraded it to the issue for the team to look at because I can't seem to be able to reproduce it. Ref: #9638

**Reply by failable** [reply]
Thanks!
